ICAFIX/hcp_fix_multi_run

```diff
@@ -126,6 +135,12 @@ opts_AddOptional '--processing-mode' 'ProcessingMode' '"HCPStyleData" (default)
 
 opts_AddOptional '--concatenate-only' 'ConcatOnlyStr' 'TRUE or FALSE' "When set to TRUE the script stops after the concatination step. e.g., for use in experimental alternative multi-run denoising" 'FALSE'
 
+opts_AddOptional '--ica-method' 'ICAmethod' 'MELODIC or ICASSO' "Use single-pass MELODIC (default) or multi-pass ICASSO consensus method for ICA" 'MELODIC'
+
+opts_AddOptional '--parallel-limit' 'parLimit' 'integer' "set how many subjects to do in parallel (local, not cluster-distributed) during individual projection and cleanup, defaults to all detected physical cores" '-1'
```

Comment:
`hcp_fix_multi_run` only runs one subject at a time, so the description of the argument here about running "subjects in parallel" doesn't make sense to me.

Reply:
It is probably supposed to be iterations.